### PR TITLE
feat: add 404 page not found site

### PR DESCRIPTION
**Pull request by danielpeintner**
fixes https://github.com/w3c/wot-marketing/issues/139

### CODE REVIEW

**Comment by danielpeintner** [comment]
It shows a rather simple 404 page for broken links like "igX". The important point is that it still shows the navigation et cetera so that someone encountering a broken link can continue to navigate.

grafik

**Review by relu91 — APPROVED** [review]

**Comment by sebastiankb** [comment]
+1

**Comment by danielpeintner** [comment]
We can wait for "official" approval next week but I would rather merge sooner than later given that 
* it helps user coming from older/broken URLs/links that might not be valid any longer
* it does not add any content that might be questionable

**Pull request merged**
